spatialdata/_core/_transform_elements.py

```diff
@@ -102,31 +109,75 @@ def _transform_raster(
         else:
             assert n_spatial_dims == 3
             # raise NotImplementedError()
-    return transformed_dask
+    return transformed_dask, translation
+
+
+def _prepend_transformation(
```

Comment:
imho this should have a docstring or more comments even if private

Reply:
done